pkgs/by-name/ra/redbean/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc  ];
+  arch =
+    if stdenv.is64bit
+    then "x86_64"
+    else "x86";
+in
+stdenv.mkDerivation rec {
+  pname = "Redbean";
+  version = "2.2";
+
+  src = fetchurl {
+    url = "https://redbean.dev/redbean-${version}.com";
+    sha256 = "db8fc7cc5a7703b7ccb830a366eb69e728fc7892fd3ecc093c089d837aa5b91b";
+  };
+
+  unpackPhase = "cp $src redbean.com";
+
+  nativeBuildInputs = [];
+
+  dontBuild = true;
+
+  dontFixup = true; # essential, otherwise cosmopolitcan libc exec-header gets corrupted
+
+  installPhase = ''
+    mkdir -p $out/bin
+    ls -la
+    mv redbean.com $out/bin/redbean.com
+    bin="$out/bin/redbean.com"
+    chmod 755 $bin
```

All of this can be consolidated in a single command.
```suggestion
    install -D $src $out/bin/redbean.com
```
`755` is the default mode in `install`, but if you want to be explicit, you can add `-m 755` to the command